projects/angular/src/accordion/stepper/stepper-panel.html

```diff
@@ -0,0 +1,64 @@
+<ng-container *ngIf="panel | async; let panel">
+  <div [ngClass]="getPanelStateClasses(panel)">
+    <div class="clr-accordion-header clr-stepper-header">
+      <button
+        type="button"
+        class="clr-accordion-header-button"
+        (click)="togglePanel()"
+        [id]="getAccordionHeaderId(panel.templateId)"
+        [disabled]="isAccordion && panel.disabled"
+        [attr.aria-disabled]="!isAccordion && panel.disabled"
+        [attr.aria-controls]="getAccordionContentId(panel.templateId)"
+        [attr.aria-expanded]="panel.open"
+        [class.clr-accordion-header-has-description]="(accordionDescription.changes | async)?.length || accordionDescription.length"
+        #headerButton
+      >
+        <span class="clr-step-status">
+          <cds-icon shape="angle" direction="right" class="clr-accordion-angle clr-step-angle"></cds-icon>
+          <cds-icon
+            status="danger"
+            *ngIf="panel.status === AccordionStatus.Error"
+            shape="exclamation-circle"
+            class="clr-step-error-icon"
+          ></cds-icon>
+          <cds-icon
+            status="success"
+            *ngIf="panel.status === AccordionStatus.Complete"
+            shape="check-circle"
+            class="clr-step-complete-icon"
+          ></cds-icon>
+        </span>
+        <span class="clr-step-title">
+          <span class="clr-step-number">{{panelNumber}}.</span>
+          <ng-content select="clr-accordion-title, clr-step-title"></ng-content>
+        </span>
```

Comment:
This should have only the ng-content  and eventually create the same selector as accordion-title.ts

Reply:
Stepper's logic separation from Accordion will be implemented with CDE-2329 in v18.